packages/SketchMorph2-Core.package/M2Backend.class/instance/penNibOffset..st

```diff
@@ -0,0 +1,3 @@
+drawing
+penNibOffset: aNumber
+	^ (aNumber asPoint / 2) floor negated
```

Comment:
```suggestion
penNibOffset: aNumberOrPoint
	^ (aNumberOrPoint asPoint / 2) floor negated
```

Reply:
Maybe it would be easier to just expect `aPoint` and cast to point before calling this method (e. g. `self brushSize asPoint`), which results in the safe assertion of the input being a point and avoids weird parameter names.